docs/source/api_reference/geodata.rst

```diff
@@ -0,0 +1,81 @@
+.. raw:: html
+    :file: embed_widgets/geodata.html
+
+GeoData
+=======
+
+``GeoData`` is an ``ipyleaflet`` class that allows you to visualize a ``GeoDataFrame`` on the Map.
+
+GeoDataFrame
+------------
+
+``GeoDataFrame`` is a tabular data structure of `Geopandas
+<http://geopandas.org/data_structures.html>`_ library. An example of ``GeoDataFrame`` :
+
+.. code::
+
+    In [1]: world = geopandas.read_file(geopandas.datasets.get_path('naturalearth_lowres'))
+
+    In [2]: world.head()
+    Out[2]:
+         pop_est      continent                      name iso_a3  gdp_md_est                                           geometry
+    0     920938        Oceania                      Fiji    FJI      8374.0  (POLYGON ((180 -16.06713266364245, 180 -16.555...
+    1   53950935         Africa                  Tanzania    TZA    150600.0  POLYGON ((33.90371119710453 -0.950000000000000...
+    2     603253         Africa                 W. Sahara    ESH       906.5  POLYGON ((-8.665589565454809 27.65642588959236...
+    3   35623680  North America                    Canada    CAN   1674000.0  (POLYGON ((-122.84 49.00000000000011, -122.974...
+    4  326625791  North America  United States of America    USA  18560000.0  (POLYGON ((-122.84 49.00000000000011, -120 49....
+
+
+Example
+-------
+
+.. code::
+
+    import ipyleaflet as ipy
```

Comment:
Do you mind doing `from ipyleaflet import Map, GeoData` for homogeneity with other examples